<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>360</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Մուտք առևտրային ծրագիր</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>400</width>
      <height>180</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="title">
     <string>Մուտքի տվյալներ</string>
    </property>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Մուտքանուն</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Գաղտնաբառ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>135</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="logInPushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>Մուտք գործել</string>
     </property>
    </widget>
    <widget class="QLabel" name="gifLabel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="show_hideButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>gifLabel</zorder>
    <zorder>show_hideButton</zorder>
    <zorder>usernameLabel</zorder>
    <zorder>usernameLineEdit</zorder>
    <zorder>passwordLabel</zorder>
    <zorder>passwordLineEdit</zorder>
    <zorder>logInPushButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
